dev/AppLifecycle/LaunchActivatedEventArgs.h

```diff
@@ -19,6 +19,11 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
         {
             m_kind = ActivationKind::Launch;
         }
+        
+        winrt::hstring GetRuntimeClassName() const noexcept override
+        {
+            return winrt::hstring(L"Windows::ApplicationModel::Activation.LaunchActivatedEventArgs"sv);
```

Comment:
These should be dots.
```suggestion
            return winrt::hstring(L"Windows.ApplicationModel.Activation.LaunchActivatedEventArgs"sv);
```

Reply:
Aren't there constants generated by MIDL for ACIDs?